lib/reversi_methods.rb

```diff
@@ -61,6 +61,7 @@ def put_stone(board, cell_ref, stone_color, dry_run: false)
   end
 
   def turn(board, target_pos, attack_stone_color, direction)
+    return false if target_pos.stone_color(board) == BLANK_CELL
     return false if target_pos.out_of_board?
     return false if target_pos.stone_color(board) == attack_stone_color
```

Comment:
`target_pos.stone_color(board)`が2回出てくるので、こんなふうにDRYにするのも一手です。
```suggestion
    return false if target_pos.out_of_board?
    target_color = target_pos.stone_color(board)
    return false if target_color == attack_stone_color || target_color == BLANK_CELL
```